functions.php

```diff
@@ -935,7 +920,7 @@ function wp_autoupdates_debug_information( $info ) {
  * @return bool True if plugins notifications are enabled, false otherwise.
  */
 function wp_autoupdates_is_plugins_auto_update_email_enabled() {
-	$enabled = ! defined( 'WP_DISABLE_PLUGINS_AUTO_UPDATE_EMAIL' ) || ! WP_DISABLE_PLUGINS_AUTO_UPDATE;
+	$enabled = true;
```

Comment:
Instead of setting `$enabled` to true, would be cleaner to just pass `true` to the filter.

Reply:
And keep mentioning $enabled in the inline docs?

Reply:
yes